.changeset/gorgeous-actors-shave.md

```diff
@@ -0,0 +1,12 @@
+---
+'@backstage/create-app': patch
```

Comment:
Breaking change for dependents of catalog-common, but not a breaking change to create-app, right?

Reply:
Yep! We've defined changes to the `create-app` template as always being `patch` changes

Reply:
Thanks for confirming, this always trips me up 😅